docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -188,3 +188,39 @@ POST _ml/set_upgrade_mode?enabled=false&timeout=10m
 DELETE temp_ml_config
 --------------------------------------------------
 // TEST[skip:TBD]
+
+[discrete]
+[[ml-debian8-memory]]
+== Suboptimal job assignment on Debian 8
+
+Where possible {ml} jobs are assigned to nodes based on the memory requirement
+of the job and the memory available on the node. However, in certain cases, the
+amount of memory on a node cannot be accurately determined, and then jobs are
+assigned by balancing numbers of jobs per {ml} node. It may lead to
+a situation where all the jobs with high memory requirements are on one node and the less memory-intensive jobs on
+another.
+
+{es} fails to determine the amount of memory on a machine that is
```

Comment:
I found the connection between the first and second paragraph to be unclear. Are these Debian 8 scenarios the only "certain cases" referred to in the first paragraph, or just one example of that? I think it would help to start the second paragraph with "For example,..." or "In particular,..." or something like that to bridge the gap.

> {es} fails